app/views/admin/schedule/show.html.haml

```diff
@@ -0,0 +1,36 @@
+.row
+  .col-md-12
+    .page-header
+      %h1 Schedule
+      %p.text-muted
+        Create the schedules for the conference
+
+.row
+  .col-md-2
+    Selected schedule
+    = check_box_tag @conference.short_title, @schedule_id, (@schedule_id == @selected_schedule_id),
+      method: :put, url: "/admin/conference/#{@conference.short_title}/schedule/#{@schedule_id}?selected_schedule=",
```

Comment:
Please use the rails url_helper and don't use string interpolation. If the route changes, your view is broken!